src/skin/qml/qmllibrarytracklistmodel.cpp

```diff
@@ -0,0 +1,85 @@
+#include "skin/qml/qmllibrarytracklistmodel.h"
+
+namespace mixxx {
+namespace skin {
+namespace qml {
+namespace {
+const QHash<int, QByteArray> kRoleNames = {
+        {QmlLibraryTrackListModel::TitleRole, "title"},
+        {QmlLibraryTrackListModel::ArtistRole, "artist"},
+        {QmlLibraryTrackListModel::AlbumRole, "album"},
+        {QmlLibraryTrackListModel::AlbumArtistRole, "albumArtist"},
+        {QmlLibraryTrackListModel::FileUrlRole, "fileUrl"},
+};
+}
+
+QVariant QmlLibraryTrackListModel::data(const QModelIndex& proxyIndex, int role) const {
+    if (!proxyIndex.isValid()) {
+        return QVariant();
```

Comment:
Let's simplify all occurrences of `return QVariant();` to `return {};`

Reply:
792acbc7dad1e6e1b0132c39e0ddb49d610a5225